### PR TITLE
from validation

### PR DESCRIPTION
**Pull request by Lynnora**

### CODE REVIEW

**Review by invisibleloop — CHANGES_REQUESTED** [review]
- All required fields need to have the `required` attribute
- The current regex pattern will only validate if you have 1 character in the field. You need to check for multiple characters and probably spaces. `[A-Za-z\s]+`
The regex above means: match `A-Z` range and/or `a-z` range `\s` spaces `+` match 1 or more of proceeding characters

File: new.html [hunk 1]
```diff
@@ -19,32 +18,32 @@ <h1 id="heading">Font Library</h1>
     </div>
     <div>
 
-<form action="index.html" method="post" enctype="multipart/form-data">
+<form action="index.html" method="post">
 <p><label for="fontname" class="label">Font Name</label>
-  <input type="text" id="fontname" name="fontname" value="" placeholder="e.g. helvectica" style="width:100%;"></p>
+  <input required title="only upper and lowercase letters allowed" pattern="[A-Za-z]" type="text" id="fontname" name="fontname" value="" placeholder="e.g. helvectica"></p>
```
Comment (invisibleloop) [inline new.html:23]: regex incorrect matches only one character

File: new.html [hunk 2]
```diff
@@ -19,32 +18,32 @@ <h1 id="heading">Font Library</h1>
...
-<p><label for="ponumber" class="label">P.O. Number</label>
-<input type="ponumber" name="ponumber" id="ponumber" style="width:30%;">
-<input type="submit" value="Upload Image" name="submit"></p>
+<p><label for="ponumber" class="label ponumber">P.O. Number</label>
+<input class="ponumber" type="ponumber" name="ponumber" id="ponumber" style="width:30%;">
+</p>
 
 <label for="requestedby" class="label">Requested By</label>
-<input type="text" id="requestedby" name="requested by" value="" placeholder="Joe Bloggs" style="width:100%" style="box-shadow:3;">
+<input type="text" id="requestedby" name="requested by" value="" placeholder="Joe Bloggs" style="box-shadow:3;">
```
Comment (invisibleloop) [inline new.html:43]: needs `required` attribute

File: new.html [hunk 3]
```diff
@@ -19,32 +18,32 @@ <h1 id="heading">Font Library</h1>
...
 
 <label for="purchaser" class="label">Purchaser</label>
-<input type="text" id="purchaser" name="purchaser" value="" style="width:100%;">
+<input type="text" id="purchaser" name="purchaser" value="">
```
Comment (invisibleloop) [inline new.html:46]: needs `required` attribute

File: new.html [hunk 4]
```diff
@@ -19,32 +18,32 @@ <h1 id="heading">Font Library</h1>
...
-  <input type="text" id="fontname" name="fontname" value="" placeholder="e.g. helvectica" style="width:100%;"></p>
+  <input required title="only upper and lowercase letters allowed" pattern="[A-Za-z]" type="text" id="fontname" name="fontname" value="" placeholder="e.g. helvectica"></p>
 
   <p><label for="foundry" class="label">Foundry</label>
-  <input type="text" id="foundry" name="foundry" value="" placeholder="linotype" style="width:100%;"></p>
+  <input required type="text" id="foundry" name="foundry" value="" placeholder="linotype"></p>
 
-  <label for="style-weight" class="label" style="width:100%;">Style-Weight</label>
+  <label for="style-weight" class="label">Style-Weight</label>
   <select  name="style-weight";>
```
Comment (invisibleloop) [inline new.html:29]: needs `required` attribute

File: new.html (hunk 1 above)
Comment (Lynnora) [reply new.html:23]: [A-Za-z0-9-]+

Comment (Lynnora) [reply new.html:23]: Should this do it?

Comment (invisibleloop) [reply new.html:23]: Ok, so here you are matching 1 or more upper and lower case characters, numbers and hyphens. Do you think you will need spaces also?

**Pull request merged**